src/apps/chifra/pkg/rpcClient/client.go

```diff
@@ -0,0 +1,28 @@
+package rpcClient
+
+import (
+	"log"
+
+	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/config"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+// Setups a client instance and returns it
+func Get() ethclient.Client {
+	client, err := ethclient.Dial(config.ReadGlobal().Settings.RpcProvider)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	return *client
+}
+
+func HexToAddress(hex string) common.Address {
```

Comment:
My idea was that because we have to import this module (`rpcClient`) whenever we want to do RPC, it would be easier to have this helper functions (that I assume we will be using often) in the same "namespace", so we don't have to import two packages (our `rpcClient` and go-ethereum's `common`)